Automattic/wp-calypso
# Jetpack UX: Change XML-RPC and REST error urls and wording

Fixes https://github.com/Automattic/dotcom-forge/issues/3603

## Proposed Changes

This PR changes the docs to point to the new docs and updates the wording of messages to match them.

## Testing Instructions


### Test 1: REST API errors
2. Create a new WoA site.
3. Enable ssh.
4. Visit eg `/posts` page
5. Login to the site and add the following to your `wp-config`
```
if ( ! empty( $_GET['rest_route'] ) ) {
	http_response_code( 404 );
	die('not found');
}
```
6. Ensure that you see the following error:
![rest_error_new](https://github.com/Automattic/wp-calypso/assets/497103/daf4a84a-64b1-4bd0-b8c3-566ac80927d6)
7. Ensure that docs point [here](https://wordpress.com/support/resolve-jetpack-errors/#rest-api-or-xml-rpc-is-blocked)

### Test 2: XML-RPC errors
1. Apply D119857-code
2. Create a new WoA site.
3. Enable ssh.
4. Visit eg `/posts` page
5. Login to the site and add the following to your `wp-config`
```
if ( ! empty( $_GET['for'] ) && $_GET['for'] === 'jetpack' ) {
	http_response_code( 404 );
	die('not found');
}
```
6. Ensure that you see the following error:
![rpc_error_new](https://github.com/Automattic/wp-calypso/assets/497103/db4cfae5-5119-45ce-9bb4-d8a29d5d6b1d)
7. Ensure that docs point [here](https://wordpress.com/support/resolve-jetpack-errors/#rest-api-or-xml-rpc-is-blocked)






## Pre-merge Checklist
